tests/test_biophysical/test_animal/test_cow_calf/test_beef_herd.py

```
@@ -0,0 +1,775 @@
+    assert hm.beef_cows is not hm.beef_replacement_heifers
+    assert hm.beef_cows is not hm.beef_calves
+    assert hm.beef_cows is not hm.beef_bulls
+
+
+# ---------------------------------------------------------------------------
+# Task 7.2 — RED tests
+# ---------------------------------------------------------------------------
+
+
+def _make_herd_manager_stub(
+    mocker: MockerFixture,
```

Ruff (0.15.20)

[warning] Unused function argument: `mocker`

(ARG001)